toolkit/resources/manifests/package/ccache-configuration.json

```diff
@@ -0,0 +1,5674 @@
+{
+  "remoteStore": {
+    "type": "azure-blob-storage",
+    "tenantId": "",
+    "userName": "",
+    "password": "",
+    "storageAccount": "marinerccache",
+    "containerName": "20-stable",
+    "tagsFolder": "tags",
+    "downloadEnabled": true,
+    "downloadFolder": "latest",
+    "uploadEnabled": false,
+    "uploadFolder": "",
+    "updateLatest": false
+  },
+  "groups": [
```

Comment:
Does this file cover every core package spec? Will we need to update it as new specs are added or removed?

Reply:
Would it be better to have a list of exceptions? Curious how many packages aren't 1:1 mappings. (ie how many are omitted, or have multiple together).

Reply:
The packages in this file are the ones where the tools will download/upload. We'd need to maintain the list every time a new package is added/removed - which is not ideal.
I agree a list of exceptions is a better option. Originally, I thought we will have a small set of packages that we want ccache download/upload enabled for to avoid size getting out of control.
After doing a full build, the entire ccache is about 30GB - which is manageable. So, we can keep the long list of packages.
I'll convert it into a list of exceptions - where:
- by default, all packages have ccache enable and they belong to their own group.
- groups that contain multiple packages will need to be defined in this file.
- groups that are disabled will need to be defined in this file.

That way, we don't need to maintain the file as much - addressing Andrew's point as well.

Reply:
@anphel31 @dmcilvaney  - I've updated to logic as follows:
- the default for any package is that ccache is enabled. It will build in its own folder by default as well.
- the config is used to:
  - disable ccache for certain packages.
  - create package groups.